source/include/microapp/cs_Microapp.h

```diff
@@ -57,6 +57,11 @@ class Microapp: public EventListener {
 	 * App state is updated in this call, make sure to store the state afterwards.
 	 */
 	cs_ret_code_t validateApp(uint8_t index);
+	
+	/**
+	 * Handle watchdog info if available for app.
+	 */
+	cs_ret_code_t handleWatchdogInfo(uint8_t index);
```

Comment:
Comment doesn't say anything about what this does.
From implementation it looks like the function name should be something like `checkWatchdogInfo()`